example_2/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+cmake_minimum_required(VERSION 3.5)
+project(ros2_control_demo_example_2)
```

Comment:
Are you happy with this name, or would you like to have it more descriptive?

I am fine with both ways. @Xi-HHHM what do you think? (check the context too)

Reply:
it's rather verbose already. Using this repo as a tutorial, then the numbered examples are fine. For using it as a reference repo some more descriptive would be better?